.github/workflows/labeler.yml

```diff
@@ -9,3 +9,10 @@ jobs:
         with:
           sync-labels: ''
           repo-token: "${{ secrets.GITHUB_TOKEN }}"
+  milestone:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: iyu/actions-milestone@v1
+        with:
+          configuration-path: .github/milestone.yml
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Great idea! Feel free to change the `labeler.yml` filename and workflow name ('Pull Request Labeler') to something more comprehensive (`pull_request_setup.yml`?).

On adding this as separate jobs (as here) vs. steps, as soon as we don't hit the [GH free plan limit (2,000 minute/month)](https://docs.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/about-billing-for-github-actions), we can still go with jobs (they'll be faster as they'll run in parallel).

Reply:
indeed. as it both adds labels *and* set milestone, i'd like to have a better and less misleading name than "Pull Request Labeler" without sacrificing  the information passed by the original name. also, to "label" something can be very general when one considers milestone yet another label. =)

but you know, to name something is difficult in computer science ...

Reply:
regarding to the quota, yes. that's the primary reason why i create a commit to merge these two actions. another reason is to have less "checks". IMHO, to complete the two of the checks is not my concern, we know, the bottle neck is always the "ceph API test" and the "make check" runs.

Reply:
also, you feel strong on the naming, how about dropping the commit merging these two actions from this PR? so we can backport the milestone action to LTS branches first.

Reply:
> indeed. as it both adds labels _and_ set milestone, i'd like to have a better and less misleading name than "Pull Request Labeler" without sacrificing the information passed by the original name. also, to "label" something can be very general when one considers milestone yet another label. =)

So what about "Pull Request Configuration"? If I'm not wrong, each of the jobs will generate separate 'checks' as follows:
* Pull Request Configuration / labeler (pull_request_target)
* Pull Request Configuration / milestone (pull_request_target)

If you want to print a single check, you'll need to move both to a single job with 2 steps.

> but you know, to name something is difficult in computer science ...

And not only in computer science, also in science fic... I meant, astrophysics. I was recently listening a podcast about how [this new term (ichnoscale)](https://arxiv.org/pdf/2103.01536.pdf) was coined.

Reply:
🤣  updated and repushed

Reply:
@epuertat just came across https://docs.github.com/en/actions/reference/authentication-in-a-workflow#using-the-github_token-in-a-workflow. i feels that "triage" is a better name than "configuration" or "setup".

Reply:
Makes a lot of sense. Like that! 👍🏼